extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/WebSocketsServerRuntimeConfig.java

```diff
@@ -39,4 +40,9 @@ public interface WebSocketsRuntimeConfig {
      */
     OptionalInt maxMessageSize();
 
+    /**
+     * The interval after which the server sends a ping message to a connected client automatically.
+     */
+    Optional<Duration> autoPingInterval();
```

Comment:
Should we add that if not set, no ping will be sent?

Reply:
Yes, we could add this to the javadoc. I thought that the return type is self-explanatory but better to be specific ;-).